src/services/auditLogService.ts

```diff
@@ -9,8 +9,8 @@ export async function createAuditLog(
     action: gqlTypes.ActionType, 
     objectType: gqlTypes.ObjectType, 
     updatedBy: string, 
-    newData: Partial<dbSchema.Submission> | gqlTypes.Submission | null,
-    oldData: Partial<dbSchema.Submission> | gqlTypes.Submission | null,
+    newData: Partial<dbSchema.Submission> | gqlTypes.Submission | string | null,
+    oldData: Partial<dbSchema.Submission> | gqlTypes.Submission | string | null,
```

Comment:
This function needs to be refactored, but wanted to do this in a separate PR since I need to change some things in the submission service if a refactor. I Already created a issue for it.